public/components/content-list-item/content-list-item.js

```diff
@@ -2,6 +2,16 @@
 const OPHAN_PATH = 'https://dashboard.ophan.co.uk/summary?path=/',
     LIVE_PATH = 'http://www.theguardian.com/';
 
+export function toInitials(str) {
```

Comment:
There may be a better place for this function live, possibly a helper library somewhere or a separate file.

Reply:
Yeah, I did think about that, since it's now used in two places. I'll have a look and see if there is an existing helpers or utils file that it would make sense in.